core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -6828,6 +6838,11 @@ private static String sums(int n, boolean c) {
       new Dependent(20, "San Diego"),
     };
 
+    public final DataType[] types = {
```

Comment:
field naming is quite odd.
Can you please just add `BigDecimal` to `org.apache.calcite.test.ReflectiveSchemaTest.EveryType` instead?

Reply:
Ok, i will make a change, thanks for your suggestion

Reply:
@vlsi changes have been made as you suggested,  could you take time to have a look again? thanks